typescript/src/api-doc-types/ParameterDoc.ts

```diff
@@ -11,7 +11,7 @@ import { ParameterContainer } from './ParameterContainer';
  */
 export class ParameterDoc extends BaseApiDoc {
   docType = 'parameter';
-  type = getDeclarationTypeText(this.declaration);
+  type = this.getTypeString(this.declaration);
```

Comment:
I'm starting with parameters but I think we probably want to roll this fix out to function return types too, and others...?

Reply:
Yeah that would be good to have. But can be a follow-up I'd say.